ripe/atlas/tools/commands/measure.py

```diff
@@ -480,6 +496,20 @@ def _get_source_kwargs(self):
             r["type"] = "msm"
             r["value"] = self.arguments.from_measurement
 
+        if self.arguments.include_tag or self.arguments.exclude_tag:
+            r["tags"] = {}
+            r["tags"]["include"] = self.arguments.include_tag or []
+            r["tags"]["exclude"] = self.arguments.exclude_tag or []
+        else:
+            smart_tags = conf["smart-tags"]
+            try:
+                r["tags"] = smart_tags[self.arguments.type][str(self.arguments.af)]
```

Comment:
Just a shorter way to say `conf["smart-tags"]`.
More comments on my idea on Issue https://github.com/RIPE-NCC/ripe-atlas-tools/issues/25#issuecomment-148123848